doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -216,190 +216,200 @@ of the following form:
-   versions, please see below:
+#. Begin by having the cluster forget the OSD. This step removes the OSD from
+   the CRUSH map, removes the OSD's authentication key, and removes the OSD
+   from the OSD map. (The :ref:`purge subcommand <ceph-admin-osd>` was
+   introduced in Luminous. For older releases, see :ref:`the procedure linked
+   here <ceph_osd_purge_procedure_pre_luminous>`.):
 
    .. prompt:: bash $
 
       ceph osd purge {id} --yes-i-really-mean-it
 
-#. Navigate to the host where you keep the master copy of the cluster's
-   ``ceph.conf`` file:
+
+#. Navigate to the host where the master copy of the cluster's
+   ``ceph.conf`` file is kept:
 
    .. prompt:: bash $
 
       ssh {admin-host}
       cd /etc/ceph
       vim ceph.conf
 
-#. Remove the OSD entry from your ``ceph.conf`` file (if it exists)::
+#. Remove the OSD entry from your ``ceph.conf`` file (if such an entry
+   exists)::
 
-	[osd.1]
-		host = {hostname}
+    [osd.1]
+        host = {hostname}
 
-#. From the host where you keep the master copy of the cluster's ``ceph.conf``
-   file, copy the updated ``ceph.conf`` file to the ``/etc/ceph`` directory of
-   other hosts in your cluster.
+#. Copy the updated ``ceph.conf`` file from the location on the host where the
+   master copy of the cluster's ``ceph.conf`` is kept to the ``/etc/ceph``
+   directory of the other hosts in your cluster.
 
-If your Ceph cluster is older than Luminous, instead of using ``ceph osd
-purge``, you need to perform this step manually:
+.. _ceph_osd_purge_procedure_pre_luminous:
 
+If your Ceph cluster is older than Luminous, you will be unable to use the
+``ceph osd purge`` command. Instead, carry out the following procedure:
 
-#. Remove the OSD from the CRUSH map so that it no longer receives data. You may
-   also decompile the CRUSH map, remove the OSD from the device list, remove the
-   device as an item in the host bucket or remove the host  bucket (if it's in the
-   CRUSH map and you intend to remove the host), recompile the map and set it.
-   See `Remove an OSD`_ for details:
+#. Remove the OSD from the CRUSH map so that it no longer receives data (for
+   more details, see `Remove an OSD`_):
 
    .. prompt:: bash $
 
       ceph osd crush remove {name}
 
+   Instead of removing the OSD from the CRUSH map, you might opt for one of two
+   alternatives: (1) decompile the CRUSH map, remove the OSD from the device
+   list, and remove the device from the host bucket; (2) remove the host bucket
+   from the CRUSH map (provided that it is in the CRUSH map and that you intend
+   to remove the host), recompile the map, and set it:
+
+
 #. Remove the OSD authentication key:
 
    .. prompt:: bash $
 
       ceph auth del osd.{osd-num}
 
    The value of ``ceph`` for ``ceph-{osd-num}`` in the path is the
-   ``$cluster-$id``.  If your cluster name differs from ``ceph``, use your
-   cluster name instead.
+   ``$cluster-$id``. However, if your cluster name differs from ``ceph``, use
```

Comment:
None of this $cluster-$id stuff matches up with the commands that we've provided.

Reply:
These variables show up in `ceph.conf`, where they were fairly obscure.  At this point I think if someone is still using a custom cluster name they aren't running Quincy or Reef anyway

Reply:
I would just remove new lines 401-404.

Reply:
I agree. Accepted, with thanks.